UmpleParser/src/ParseUtilities_Code.ump

```diff
@@ -0,0 +1,606 @@
+/*
+
+Copyright: All contributers to the Umple Project
+
+This file is made available subject to the open source license found at:
+http://umple.org/license
+
+ */
+
+class Token
+{
+  depend java.util.regex.*;
+  public Token(String aName, String aValue, Position aPosition)
+  {
+    this(aName,aValue);
+    setPosition(aPosition);
+  }
+
+  public static boolean isValidIdentifier(String identifier, String prefix){
+    String _prefix = "^" + prefix;
+    String body = "[a-zA-Z0-9_]*$";
```

Comment:
**If** this regex can accept spaces, it could be simplified to: `[\w_]*$`, when used in Java: `"[\\w_]*$"`.